styles.css

```diff
@@ -156,18 +159,30 @@ h2 {
 .img1 {
     aspect-ratio: auto 212 / 54;
 }
-#mail {
-    border-style: none;
-    border-radius: 25px;
+.inputwrapper {
+    width: 100%;
```

Comment:
It couldn't be 100% since for devices with width less than approximately 550px it will not fit. (You forgot about margins so size should be 100% - all margins to fit the screen)